client/client-lib/src/mint/mod.rs

```diff
@@ -251,18 +248,34 @@ impl MintClient {
         dbtx.commit_tx().await.expect("DB Error");
     }
 
+    pub async fn set_notes_per_denomination(&self, notes: u16) {
+        let mut dbtx = self.start_dbtx().await;
+        dbtx.insert_entry(&NotesPerDenominationKey, &notes)
+            .await
+            .expect("DB error");
+        dbtx.commit_tx().await.expect("DB error");
+    }
+
+    async fn notes_per_denomination(&self, dbtx: &mut DatabaseTransaction<'_>) -> u16 {
```

Comment:
nit: Should probably be `pub` if a UI wants to display the current value in settings.